deploy/migrations.config.ts

```diff
@@ -1,5 +1,32 @@
 import {ethers} from "ethers"
 
+const geth = {
```

Comment:
Maybe we should call it `dev` instead of `geth`? Or does `geth` always mean the local dev network?

Reply:
Initially I was undecided on whether just use the [localhost network](https://github.com/livepeer/protocol/blob/confluence/hardhat.config.ts#L85). But I guess a combination of the two works fine - `gethDev`. (and no, geth != local dev network)

Reply:
`gethDev` is ok. Also `dev` or `devnet` would work imo.